docs/unique/ai-performance-optimization.md

```
@@ -0,0 +1,94 @@
+
+- Additional Improvements:
+
+- Set up kitchen sink compose file
+  - Has vLLM for optimized batched LLM queries
+  - Has OpenVINO GenAI model server (NPU)
+
+[[vLLM]] [[OpenVINO]]
+
+#AI #Optimization #Parallelization #BatchProcessing #Memoizer #PromptEngineering
+<!-- GENERATED-SECTIONS:DO-NOT-EDIT-BELOW -->
+## Related content
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)